Drshyamsundaram patch 2

drshyamsundaram opened the pull request:
DEPA Definitions section added. Request review and closure

drshyamsundaram committed:
Create Ddepa_definitions.md

DEPA Definitions (without formula). Document under review with ISPIRT team

drshyamsundaram committed:
Rename Ddepa_definitions.md to depa_definitions.md

Renamed the file for accurate name

drshyamsundaram committed:
Update depa_definitions.md

Added base pre-definitions

drshyamsundaram committed:
Update depa_definitions.md

Minor changes to the SRO example aspects

drshyamsundaram committed:
Create depa_definitions_mapper.md

DEPA definitions mapper

drshyamsundaram committed:
Create depa_definitions_mapper.md

Added depa_definitions_mapper

drshyamsundaram committed:
Delete internal/specification/depa_definitions_mapper.md

Delete this file. Have added this to the new subfolder under definitions

drshyamsundaram committed:
Create depa_usecase_inference_dpdp_template.md

Use Case template added

kapilvgit commented on internal/definitions/depa_definitions.md:
We already have https://depa.world/training/glossary. Adding another page might be confusing. Can we merge in some way?

drshyamsundaram replied:
understand. Are you ok with including those not covered in glossary to be added in definitions.

drshyamsundaram committed:
Added depa_definitions to glossary alomg with terms addition in prior glossary.

drshyamsundaram committed:
Added section 1.3 for expansions. The flow is required to main the logic and terms.

drshyamsundaram committed:
Review version with all retrofit done.

drshyamsundaram committed:
moved all glosaary to section 1.3 to keep it consistent. All definitions are in syc.

drshyamsundaram committed:
Create functional_specs.md

This is quick functional specifications to be used for sharing with DEPA orientation in a quick view.

drshyamsundaram committed:
Update depa_definitions.md

Sorted formatting Issues. To be merged with glossary.md

drshyamsundaram committed:
Update depa_definitions.md

All format and content restructure done.

drshyamsundaram committed:
all revisions and changes for glossary.md replacement done

drshyamsundaram commented:
Updating the glossary.md as discussed with format feedback’s.
